pkg/terraform/state/handler.go

```diff
@@ -137,9 +154,25 @@ func (h *handler) OnChange(key string, obj *v1.State) (*v1.State, error) {
 	v1.StateConditionJobDeployed.True(obj)
 	obj.Status.ExecutionName = exec.Name
 	obj.Status.LastRunHash = runHash
+
+	if obj.Annotations[RefreshAnnotationKey] != "" {
+		duration, err := time.ParseDuration(obj.Annotations[RefreshAnnotationKey])
+		if err != nil {
+			logrus.Error(err)
```

Comment:
Perhaps a more descriptive error message? Explicitly highlighting that annotation value was invalid? Assuming of course the stacktrace is quite verbose and difficult to parse?

Reply:
@donovanmuller better now?

Reply:
@k0da assuming you referring too [line 94](https://github.com/AbsaOSS/terraform-controller/pull/15/commits/80fecf22eff5cf91c4b69195570681adebaae73c#diff-d6000bd7a1be9e2afee77969b28e9552ca25a7a548818aef7fe4bf49ce348269R94) then yes, happy

Reply:
@k0da, thx, please, can you also fix here?

Reply:
oh, right, done